### PR TITLE
Eduedition

### PR DESCRIPTION
**Pull request by jospoortvliet**
Update website for 12.0.1 & Nc EduEdition

### CODE REVIEW

**Comment by jospoortvliet** [comment]
Has already been reviewed by Marius.

File: page-education.php [hunk 1]
```diff
@@ -67,6 +68,58 @@
...
+        </div>
+        <div class="col-sm-4 app-feature">
+            <img class="img-responsive revealOnScroll" src="<?php bloginfo('template_directory'); ?>/assets/img/features/zenodo-nw.png" />
+            <h2 class="section--heading-2 text-center revealOnScroll">Zenodo</h2>
+            <p class="section--paragraph revealOnScroll"><?php echo $l->t('The Zenodo Publishing app enables researchers to publish papers on the popular Zenodo.org platform, developed and supported by European Science Accellerators CERN.');?></p>
+        </div>
+        <div class="col-sm-4 app-feature">
+            <img class="img-responsive revealOnScroll" src="<?php bloginfo('template_directory'); ?>/assets/img/features/orcid-nw.png" />
+            <h2 class="section--heading-2 text-center revealOnScroll">ORCID & SAML</h2>
+            <p class="section--paragraph revealOnScroll"><?php echo $l->t('Educational organizations often use SAML, Shibboleth, Kerberos and ORCID authentication. These and more authentication mechanisms are available in the Education Edition.');?></p>
```
Comment (MorrisJobke) [inline page-education.php]: ORCID is not an auth mechanism for Nextcloud, but this is needed to auth against the Zenodo backend from within Nextcloud. ORCID provides a unique ID for author. It is used by zenodo to identify authors when publishing their work.

@jospoortvliet

File: page-education.php [hunk 2]
```diff
@@ -67,6 +68,58 @@
...
+            <p class="section--paragraph revealOnScroll"><?php echo $l->t('The Moodle integration offers users the ability to access files from Nextcloud, adding, removing or sharing them from within Moodle. More features are in development through a collaboration between Moodle Pty Ltd and Nextcloud GmbH.');?></p>
+        </div>
+        <div class="col-sm-4 app-feature">
+            <img class="img-responsive revealOnScroll" src="<?php bloginfo('template_directory'); ?>/assets/img/features/zenodo-nw.png" />
+            <h2 class="section--heading-2 text-center revealOnScroll">Zenodo</h2>
+            <p class="section--paragraph revealOnScroll"><?php echo $l->t('The Zenodo Publishing app enables researchers to publish papers on the popular Zenodo.org platform, developed and supported by European Science Accellerators CERN.');?></p>
+        </div>
+        <div class="col-sm-4 app-feature">
+            <img class="img-responsive revealOnScroll" src="<?php bloginfo('template_directory'); ?>/assets/img/features/orcid-nw.png" />
+            <h2 class="section--heading-2 text-center revealOnScroll">ORCID & SAML</h2>
```
Comment (nickvergessen) [inline page-education.php]: is this correct, or should it be `&amp;`

Comment (MorrisJobke) [inline page-education.php]: It should be `&amp;`, but browsers know that humans do errors and render it properly if it is not followed by the correct escape sequence.

**Pull request merged**